### ISSUE TITLE
Error occurred when finalizing GeneratorDataset iterator: Failed precondition: Python interpreter state is not                                                                                                                                           initialized. 

### ISSUE THREAD

**Issue by zaheerbeg21** [Adeel-Intizar/Xtreme-Vision#7]
I install xtreme-vision
pip install --no-dependencies xtreme-vision
it shows version - xtreme_vision - 1.5.1

On colab it runs with only - pip install Xtreme-Vision 
python version as - 
Python 3.6.9 (default, Apr 18 2020, 01:56:04)
[GCC 8.4.0] on linux

but on linux server it does not 
OS. Version as - 
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.3 LTS
Release:        18.04
Codename:       bionic

I installed all this packages on public ip separately  - 

tensorflow  - 2.3.0-dev20200522	
keras - 2.4.3
PIL - 8.0.1
cv2 - 4.5.1-pre
matplotlib - 3.3.3
numpy - 1.18.4
pandas - 1.1.5
sklearn - 0.23.1
imgaug - 0.4.0
labelme2coco - 0.1.2
progressbar - 2.3dev
scipy - 1.4.1
h5py - 2.10.0
matplotlib - 3.3.3
skimage - 0.17.2
after running train_yolov4.py 
after 45 mis shows Error as - >
===============================
Errors may have originated from an input operation.
Input Source operations connected to node gradient_tape/YOLOv4/CSPDarknet53/csp_res_net_1/yolo_conv                                                                                                                                          2d_16/sequential_18/mish_16/Sigmoid:
 YOLOv4/CSPDarknet53/csp_res_net_1/yolo_conv2d_16/sequential_18/batch_normalization_16/FusedBatchNo                                                                                                                                          rmV3 (defined at /Xtreme-Vision/xtreme_vision/Detection/yolov4/model/common.py:92)

Function call stack:
train_function

2020-12-11 00:32:53.589305: W tensorflow/core/kernels/data/generator_dataset_op.cc:103] Error occurred when finalizing GeneratorDataset iterator: Failed precondition: Python interpreter state is not                                                                                                                                           initialized. The process may be terminated.
         [[{{node PyFunc}}]]

Kindly guide please...

**Comment by Adeel-Intizar** [comment]
This package has only been tested on windows currently, but if you install it properly it should work.
Try uninstalling it and all the dependencies as well, and then type the following command, it will automatically download all the dependencies, you don't have to install them by yourself because it may unstable the package. 

try to install it simply with 

!pip install xtreme-vision
or
!pip install -q xtreme-vision (in case if you don't want to show the output of installation process)

**Comment by zaheerbeg21** [comment]
can we use it with tensorflow 2. 1.o ? what should i change to run it on
tensorflow 2. 1, please suggest something brother
I want to run it on watson machine learning community edition container
which has built in  tensorflow 2. 1

**Comment by Adeel-Intizar** [comment]
it can be used on any tensorflow version until it is above 2.0 but there might be some unstability

**Comment by Adeel-Intizar** [comment]
but tensorflow 2.3 is recommended
